.github/workflows/build.yaml

```diff
@@ -64,41 +64,81 @@ jobs:
              make_args="${make_args} image=${{ env.image }}"
          fi
          echo "make_args=${make_args}" >> $GITHUB_ENV
-         c_image_host=$(echo ${{ env.image }} | cut -d '/' -f 1 | rev | cut -d '.' -f 1-2 | rev)
-         case ${c_image_host} in
-             docker.com | docker.io)
-                 d_registry="docker.io"
-                 d_username=${{ secrets.DOCKER_USERNAME }}
-                 d_password=${{ secrets.DOCKER_TOKEN }}
-                 ;;
-             ghcr.io)
-                 d_registry="ghcr.io"
-                 d_username="token"
-                 d_password=${{ secrets.GHCR_PUSH_TOKEN}}
-                 ;;
-             *)
-                 d_registry=${c_image_host}
-                 d_username=''
-                 d_password=''
-                 ;;
-         esac
-         echo "d_registry=${d_registry}" >> $GITHUB_ENV
-         echo "d_username=${d_username}" >> $GITHUB_ENV
-         echo "d_password=${d_password}" >> $GITHUB_ENV
+         split_images=$(echo ${{ env.image}} | tr ',' ' '
```

Comment:
Can you add a comment here that you are creating per-service variables from a comma-separated list of images?

Also inconsistent spacing around `env.image`...

Reply:
Done.